secretmanager/snippets/regional_samples/snippets_test.py

```diff
@@ -278,6 +286,84 @@ def test_get_regional_secret_version(
     assert secret_id in version.name
     assert version_id in version.name
 
+
+def test_list_regional_secret_versions(
+    capsys: pytest.LogCaptureFixture,
+    regional_secret_version: Tuple[str, str, str, str, str],
+    another_regional_secret_version: Tuple[str, str, str, str, str],
+) -> None:
+    project_id, location_id, secret_id, version_id, _ = regional_secret_version
+    version_1 = get_regional_secret_version(
+        project_id, location_id, secret_id, version_id
+    )
+    _, _, _, another_version_id, _ = another_regional_secret_version
+    version_2 = get_regional_secret_version(
+        project_id, location_id, secret_id, another_version_id
+    )
+    list_regional_secret_versions(project_id, location_id, secret_id)
+
+    out, _ = capsys.readouterr()
+    assert secret_id in out
+    assert f"Found secret version: {version_1.name}" in out
+    assert f"Found secret version: {version_2.name}" in out
+
+
+def test_list_regional_secret_versions_with_filter(
+    capsys: pytest.LogCaptureFixture,
+    regional_secret_version: Tuple[str, str, str, str, str],
+    another_regional_secret_version: Tuple[str, str, str, str, str],
+) -> None:
+    project_id, location_id, secret_id, version_id, _ = regional_secret_version
+    enabled = get_regional_secret_version(
+        project_id, location_id, secret_id, version_id
+    )
+    _, _, _, another_version_id, _ = another_regional_secret_version
```

Comment:
Looks inefficient to me.

Reply:
PTAL @ https://github.com/GoogleCloudPlatform/python-docs-samples/pull/12481#discussion_r1732716453

Reply:
Fixed, will be resolved once child branch is merged.